src/main/scala/edu/ie3/simona/model/participant/evcs/EvcsChargingProperties.scala

```diff
@@ -39,7 +39,7 @@ trait EvcsChargingProperties {
   ): Power = {
     val evPower = currentType match {
       case ElectricCurrentType.AC =>
-        ev.pRatedAc
+        ev.sRatedAc
```

Comment:
EV will require in the end some ative power (as it was with pRatedAc before). Now we have only sRated provided by the API. Thus we need to adapt this and include the cosPhi here as well.